### PR TITLE
Add React Routers

### PR DESCRIPTION
**Pull request by techEmprez**
The following took place:

- Used React Router.
- Used styles in a React app.
- Created a website consisting of 3 pages: Home, Calculator, and Quote

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './Calculator.css';
+
```
Comment (codepantha) [inline src/components/Header.js]: - [ ] There seems to be a problem with this import right here as the name of the file is `calculator.js` and not `Calculator.js`. Please modify the import to match the actual file name in order to get rid of the error below 👇 
image

File: src/components/Quote.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const Quote = () => (
+  <div style={{
+    width: '60%',
+    margin: '4% auto',
+    fontSize: '1.1rem',
+  }}
```
Comment (codepantha) [inline src/components/Quote.js]: - [ ] It is not good practice to have your styling right here in the JSX. You can create a Quote.css file or simply write the styling in the App.css file instead 👍

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi @techEmprez 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Used react router
✔️ Good commit history
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
## Hello @techEmprez 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**